lib/msal-common/src/authority/Authority.ts

```diff
@@ -55,6 +58,14 @@ export class Authority {
     protected performanceClient: IPerformanceClient | undefined;
     // Correlation Id
     protected correlationId: string | undefined;
+    // Reserved tenant domain names that will not be replaced with tenant id
+    private static reservedTenantDomains: Set<string> = (new Set([
```

Comment:
nit: The first two are replacement strings and not technically part of the `authority`. The next three are STS understood `authority strings`. Can we separate them conceptually?

Reply:
The only purpose of this Set is to filter out names that should not be replaced with tenant id. I am not sure it would be better to split it into pars.